solr/modules/ltr/src/java/org/apache/solr/ltr/SolrQueryRequestContextUtils.java

```diff
@@ -54,18 +57,22 @@ public static LTRScoringQuery[] getScoringQueries(SolrQueryRequest req) {
   }
 
   /** isExtractingFeatures flag accessors */
-  public static void setIsExtractingFeatures(SolrQueryRequest req) {
-    req.getContext().put(IS_EXTRACTING_FEATURES, Boolean.TRUE);
+  public static void enableFeatureLogging(SolrQueryRequest req) {
+    req.getContext().put(IS_LOGGING_FEATURES, Boolean.TRUE);
   }
 
-  public static void clearIsExtractingFeatures(SolrQueryRequest req) {
-    req.getContext().put(IS_EXTRACTING_FEATURES, Boolean.FALSE);
+  public static boolean isLoggingFeatures(SolrQueryRequest req) {
+    return Boolean.TRUE.equals(req.getContext().get(IS_LOGGING_FEATURES));
   }
 
-  public static boolean isExtractingFeatures(SolrQueryRequest req) {
-    return Boolean.TRUE.equals(req.getContext().get(IS_EXTRACTING_FEATURES));
+  public static void logAllFeatures(SolrQueryRequest req, Boolean logAll) {
```

Comment:
setLogAllFeatures?

Reply:
this one I am not sure, I think the name can be reasonable without the explicit set